package.json

```diff
@@ -61,8 +61,9 @@
     "country-data": "0.0.31",
     "es-main": "^1.0.2",
     "express": "^4.18.1",
+    "ffmpeg-core-node": "^0.10.0",
+    "ffmpeg-node-patch": "^0.10.1",
```

Comment:
Link to the source of these.

Reply:
https://www.npmjs.com/~hardshutdown

Reply:
Need to see the source code though. How about you push it to a fork?

Reply:
Sorry about that, I just got off my PC. Could I provide the tarballs to you? I made a straight-up node_modules edit and published it.

Reply:
Ah I see, not a problem. I'll get the archive from npm.

Reply:
Additionally, in ffmpeg-core-node, the code was pre-built. I modified around line 250 char 35 and I put a comment around the old code, replacing it with a require to node:fs/promises. Just so you don't think I slipped anything into the minified code ;D